.travis.yml

```diff
@@ -1,28 +0,0 @@
----
-language: node_js
-node_js:
-  - "12"
-
-sudo: false
-dist: trusty
-
-addons:
-  chrome: stable
-
-cache:
-  directories:
-    - $HOME/.npm
-
-env:
-  global:
-    # See https://git.io/vdao3 for details.
-    - JOBS=1
-
-before_install:
-  - npm install -g npm@6
-  - npm config set spin false
-
-script:
-  - npm run lint
-  - npm test
-  - npx ember build -e production
```

Comment:
Unused script. (Netlify watches the repo for a successful build to the default branch.)